src/lib/uTop_private.ml

```diff
@@ -9,6 +9,28 @@
 
 open Lwt_react
 
+module DefaultPaths = struct
+  let ( / ) = Filename.concat
+  let xdg = Xdg.create ~env:Sys.getenv_opt ()
+
+  (* Fallback to legacy filename *)
+  let resolve legacy fn =
+    if not (Sys.file_exists fn) && Sys.file_exists legacy then
```

Comment:
I think we can simplify a bit the logic here to "if legacy exists, use it. otherwise, use new path". We'll add a deprecation warning when the legacy path is loaded in a couple releases.